package.json

```diff
@@ -72,9 +72,9 @@
     "fix": "eslint --color --fix --ext .ts . && tsc || true",
     "postinstall": "replace-in-file \"declare var console\" \"//declare var console\" node_modules/@types/google-apps-script/google-apps-script.base.d.ts"
   },
-  "repository": "github:grahamearley/FirestoreGoogleAppsScript",
+  "repository": "github:abkarino/FirestoreGoogleAppsScript",
```

Comment:
Yeah, already mentioned it at the top of my PR, I missed it.
I made the pull so people can continue from there. I was focusing on the functionality and had to reset my repo because the readme changes were in the same commit.